Makefile

```diff
@@ -138,7 +133,7 @@ docs/assets/diagram.png: go-diagrams/diagram.dot
 	cd go-diagrams && dot -Tpng diagram.dot -o ../docs/assets/diagram.png
 
 .PHONY: start
-start: image
```

Comment:
I decoupled building the image from starting the infra so that `kind load docker-image` can be called in between the two steps [here](https://github.com/argoproj/argo-events/pull/2207/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR138-R139).